.github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml

```diff
@@ -90,7 +90,7 @@ jobs:
         [int]$BuildIncrementer = "${{ env.BUILD_INCREMENTER }}"
         [int]$newRevision =  $BuildCounter + $BuildIncrementer
         [string]$version = "${{ env.VERSION_MAJOR}}"+ "." +"${{ env.VERSION_MINOR}}" + "." + $newRevision.ToString()
-        .\build.github.ps1 -Engine PostgreSQL -PackageVersion $version -RepositoryRoot $env:GITHUB_WORKSPACE -UsePlugins -RunSdkGen -GenerateSdkPackages -RunPostman -RunPester -RunDotnetTest -NoPackaging
+        .\build.github.ps1 -Engine PostgreSQL -PackageVersion $version -RepositoryRoot $env:GITHUB_WORKSPACE -UsePlugins -RunSdkGen -GenerateApiSdkPackage -GenerateTestSdkPackage -RunPostman -RunPester -RunDotnetTest -NoPackaging
```

Comment:
Existing teamcity build is running with GenerateApiSdkPackage  only  . Is there any reason to include also GenerateApiSdkPackage ?   I am referring this build https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_OdsApiInitDevIntegrationTest?branch=%3Cdefault%3E&buildTypeTab=overview&mode=builds